Assets/Scripts/BulletTypes/MediumBulletController.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MediumBulletController : BulletController 
+{
+    public MediumBulletController()
```

Comment:
refactor all controllers to have the common functionality in the bullet controller base class

Reply:
Done